host.go

```diff
@@ -210,12 +210,14 @@ func (h *gpbftRunner) Start(ctx context.Context) (_err error) {
 					// does, error loudly since the chances are the cause is a programmer error.
 					return errors.New("cert store subscription to finalize tipsets was closed unexpectedly")
 				}
-				key := cert.ECChain.Head().Key
-				if err := h.ec.Finalize(h.runningCtx, key); err != nil {
-					// There is not much we can do here other than logging. The next instance start
-					// will effectively retry checkpointing the latest finalized tipset. This error
-					// will not impact the selection of next instance chain.
-					log.Error(fmt.Errorf("error while finalizing decision at EC: %w", err))
+				if h.manifest.EC.Finalize {
+					key := cert.ECChain.Head().Key
+					if err := h.ec.Finalize(h.runningCtx, key); err != nil {
+						// There is not much we can do here other than logging. The next instance start
+						// will effectively retry checkpointing the latest finalized tipset. This error
+						// will not impact the selection of next instance chain.
+						log.Error(fmt.Errorf("error while finalizing decision at EC: %w", err))
```

Comment:
I fat-fingered this line, apologies. Below should fix it.
```suggestion
						log.Errorf("error while finalizing decision at EC: %w", err)
```

Reply:
Needs to be `%+v`, committing locally.